keyboards/planck/keymaps/vxid/keymap.c

```diff
@@ -0,0 +1,105 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+enum planck_layers {
+  _QWERTY,
+  _LOWER,
+  _RAISE,
+  _ADJUST
+};
+
+enum planck_keycodes {
```

Comment:
If you're not using any of these keycodes, you could remove the entire enum, actually.